skin.css

```diff
@@ -1,13 +1,50 @@
+/* css-skin-prabesh */
+
+
 .highlight {
     background-color: #ddd;
 }
 
+/* common style for all buttons with class btn */
 button.btn {
+  font-size: 1.09em;
   border-radius: 3px;
+  color: #2A31BB;
+  background: transparent;
+  padding: 14px 61px;
   border-style: solid;
   border-width: 1px;
   font-weight: normal;
   line-height: 1.15;
   text-align: center;
   vertical-align: middle;
+  margin-left: 7px;
+  outline: none;

```

Comment:
This is somewhat debated among frontend, as most people find the outline to be ugly (mostly because it is!).

However it is quite important for accessibility, as keyboard-only users need to be able to easily tell which element is focused when they are using the Tab key to navigate around the page. You can try this yourself to see what I mean, sometimes it's hard to tell where the focus is.

Reply:
Thnx 40thieves for your feekback.